### ISSUE TITLE
[Keyboard Navigation -  Alert rules] 261598: Info button (i) present for "Search" field is not keyboard accessible.

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#66467]
### User Experience
Keyboard-dependent users will not be able to access the controls and will not be able to perform the required action on the controls if interactive controls present on any page or pane are not keyboard accessible. It will block the user from getting the essential information about the respective control.

### Repro Steps
Open #Url: https://play.grafana.org/ Sign in with valid credentials. "Home" page appears. Navigate to the left navigation bar and to the "Alerting" button, select it. It gets expanded and navigates to the expanded sub menu items. Select the "Alert Rules" menu items. A new page "Alerting" appears. Navigate the tab " Alert Rules". Navigate to "Search" field. Verify the issue

 ### Actual Result:
 Issue: Info button (i) present for "Search" field is not keyboard accessible.
Observation: While navigating to "Search" text box edit field present under "Alert rules" tab and pressing f1/fn+f1/Ctrl+f1/alt f1, the Info button is not getting activated/expanded. However, it is accessible with mouse hover.

 ### Expected Result:
 Info button (i) present for "Search" field should be keyboard accessible. On navigating using keyboard focus should move to the control and it should be accessible using keyboard shortcut keys.
 
Note: The same issue exists with the same type of control info(i) throughout the application.

### Severity
`SEV2`

### Area
Alert Rules (HoverCard Component)

### MS Sheet ID
[261598](https://docs.google.com/spreadsheets/d/1kB-Sz3lvfje93HLVIx4UhDvwgU-J7k1PVN70ryQkc4M/edit#gid=1913685923&range=A23)

https://user-images.githubusercontent.com/6906380/231772043-8f29a7ad-ce88-433d-9303-7a3ea3a47fcf.mov
